ui/contextmenuextension.cpp

```diff
@@ -118,17 +118,17 @@ void ContextMenuExtension::populateMenu(QMenu *menu)
     if (m_id.isNull())
         return;
 
-    auto probeController = ObjectBroker::object<ProbeControllerInterface *>();
-    probeController->requestSupportedTools(m_id);
+    auto toolManager = ObjectBroker::object<ToolManagerInterface *>();
+    toolManager->requestToolsForObject(m_id);
 
     // delay adding actions until we know the supported tools
-    connect(probeController, &ProbeControllerInterface::supportedToolsResponse,
+    connect(toolManager, &ToolManagerInterface::toolsForObjectResponse,
             menu, [menu](ObjectId id, const ToolInfos &toolInfos) {
         foreach (const auto &toolInfo, toolInfos) {
-            auto action = menu->addAction(tr("Show in \"%1\" tool").arg(toolInfo.name));
+            auto action = menu->addAction(tr("Show in \"%1\" tool").arg(toolInfo.id));
```

Comment:
yes, I saw this, when rebasing. I decided, that fixing it is out of scope for this change... I'd rather do it in a separate PR.